components/cards/CountdownCard.jsx

```diff
@@ -0,0 +1,187 @@
+import React, { useState, useRef, useEffect } from "react";
+import DatePicker from "react-datepicker";
+import Image from "next/image";
+import editIcon from "../../public/icons/edit.svg";
+import styled from "styled-components";
+import { CreateCountdownTitle } from "../cardTitle/CreateCountdownTitle";
+
+export default function CountdownCard() {
+  const [pickedDate, setPickedDate] = useState(new Date());
+  const [countdownDays, setCountdownDays] = useState("00");
+  const [editMode, setEditMode] = useState(false);
+  const [cardTitle, setCardTitle] = useState("");
+  const [displayMode, setDisplayMode] = useState(false);
+  const [editTitleMode, setEditTitleMode] = useState(true);
+  const [displayTitleMode, setDisplayTitleMode] = useState(false);
+
+  let interval = useRef();
+  const startCountdown = () => {
+    const countdownDate = new Date(pickedDate.toString()).getTime();
+    interval = setInterval(() => {
+      const now = new Date().getTime();
+      const distance = countdownDate - now;
+
+      const days = Math.floor(distance / (1000 * 60 * 60 * 24));
+
+      if (distance < 0) {
+        //stop the countdown
+        clearInterval(interval.current);
+      } else {
+        //continue the countdown
+        setCountdownDays(days);
+      }
+    }, 100);
```

Comment:
this seems to work. I wonder if the interval needs to fire every 100ms if it only shows the amount of days remaining. 
Maybe even `60000` to fire every minute should be sufficient.

Reply:
make sense. thank you for the input